<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_op1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>false</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>true</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_operation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>121</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>⋀</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>∨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>¬</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>→</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>↔</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>⊕</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_op2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>101</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>false</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>true</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
